apps/block_scout_web/assets/css/_layout.scss

```diff
@@ -13,4 +13,32 @@
 
 .logs-hash {
   line-height: 24px !important;
-}
\ No newline at end of file
+}
+
+.outdated-network-modal-container {
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  z-index: 999;
+}
+
+.outdated-network-modal-bg {
+  background: #000;
+  width: 100%;
+  height: 100%;
+  opacity: 0.5;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.outdated-network-modal {
+  width: 100%;
+  max-width: 550px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translateX(-50%) translateY(-50%);
+}
```

Comment:
We should try to avoid modifying this file to reduce conflicts with the main repo.

Can't these customizations be in `theme/_polygon_hermez_variables.scss`?

Also, this file should probably be renamed as `_polygon_zkevm_variables.scss`.

Reply:
right, done!